packages/tailwindcss/src/compat/flatten-color-palette.ts

```diff
@@ -0,0 +1,19 @@
+type Colors = {
+  [key: string | number]: string | Colors
+}
+
+export function flattenColorPalette(colors: Colors) {
```

Comment:
TODO: We need to expose this to the end user for backwards compatibility. I believe it's `tailwindcss/lib/util/flattenColorPalette`

Reply:
Really unfortunate it needs to live at `tailwindcss/lib/util/flattenColorPalette` and not `tailwindcss/flattenColorPalette`.

Do you think we should do this in this PR or a followup?

Reply:
Can that be something we configure in our exports config or copy over at build-time or something I wonder?

Reply:
I think we would still require a separate file to point to, but the paths should be fixable via exports I believe. 

I also think that it should be a default export in the final result so that you can do `const flattenColorPalette = rqeuire('...')` which is what's currently used.